src/sicmutils/env.cljc

```diff
@@ -48,6 +48,7 @@
             [sicmutils.numerical.quadrature]
             [sicmutils.mechanics.lagrange]
             [sicmutils.mechanics.hamilton]
+            [sicmutils.mechanics.rigid]
```

Comment:
this is a very cool namespace behind Colin's latest "tumbling T handle" demo that I want to recreate. We don't explicitly bind any of its symbols in `env` but I want it available for requiring.